connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -103,6 +103,12 @@ public String protocolType() {
         return "connect";
     }
 
+    // expose for tests
+    @Override
+    protected synchronized boolean ensureCoordinatorReady(final long timeoutMs) {
```

Comment:
Do we need this? Seems like the same signature as the parent.

Reply:
It's weird, but doing this makes the method accessible in the `WorkerCoordinatorTest`, which it isn't otherwise because the test is in a different package from `AbstractCoordinator`. `WorkerCoordinator` can see the superclass's method because it is a subclass.

Reply:
Makes sense.